### PR TITLE
add custom domain docs

### PR DESCRIPTION
**Pull request by petrkalos**

### CODE REVIEW

**Comment by noah-paige** [comment]
Doc changes look good but think we may want to merge to `gh-pages-v2m6m1` here

cc: @petrkalos @dlpzx - lets quickly align on this tomorrow?

**Comment by dlpzx** [comment]
the branch is still wring, we should put it in `gh-pages-v2m6m1` if any

File: pages/deploy/deploy_aws.md [hunk 1]
```diff
@@ -223,41 +223,43 @@ have listed and defined all the parameters of the cdk.json file. If you still ha
...
+| custom_domain                                 | Optional*                              | Custom domain configuration: `hosted_zone_name`, `hosted_zone_id`, `certificate_arn`, and `email_notification_sender_email_id`. If internet_facing parameter is **false** or `share_notifications.email` is active in `config.json` then custom_domain is REQUIRED for ECS ALB integration with ACM and HTTPS. It is optional when internet_facing is true.                                                          |
+| custom_domain                                 | [Strongly Recommended](#Enforce-TLS12) | Custom domain configuration for the frontend: `hosted_zone_name`, `hosted_zone_id`, `certificate_arn`, and `email_notification_sender_email_id`. If internet_facing parameter is **false** or `share_notifications.email` is active in `config.json` then custom_domain is REQUIRED for ECS ALB integration with ACM and HTTPS. It is optional when internet_facing is true.                                         |
```
Comment (noah-paige) [inline pages/deploy/deploy_aws.md:249]: duplicate `custom_domain` - keep the Strongly Recommended one and to delete Optional

Comment (petrkalos) [reply pages/deploy/deploy_aws.md:249]: ops good catch

**Review by noah-paige — APPROVED** [review]

**Pull request merged**